jeos-firstboot-enroll

```diff
@@ -67,6 +70,20 @@ enroll_systemd_firstboot() {
 		d_styled --yesno $"Unlock encrypted disk via TPM?" 0 0 && with_tpm2="$has_tpm2"
 	fi
 
+	if [ -n "$with_tpm2" ] && d_styled --yesno $"With TPM PIN protection?" 0 0; then
+		while true; do
+			d --insecure --passwordbox  $"Enter new PIN (actually just passphrase)" 0 0
```

Comment:
there is a lot of y/n password, what about that if the PIN is empty, no PIN is set? The text can be "Enter blank PIN to automatic unlock" or something like that.

Reply:
good idea

Reply:
is now a separate item in the menu